src/plugins/lua/known_senders.lua

```diff
@@ -197,6 +219,57 @@ local function known_senders_callback(task)
   end
 end
 
+local function check_known_incoming_mail_callback(task)
+  local sender = task:get_reply_sender()
+  if not sender then
+    rspamd_logger.errx(task, 'Couldn\'t get sender')
```

Comment:
It should not be logged as an error, maybe better just add a `info` or even `debugm` here.